Dockerfile

```diff
@@ -90,25 +90,25 @@ ENV UBSAN_OPTIONS=print_stacktrace=1
 RUN wget https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64.deb
 RUN dpkg -i dumb-init_*.deb && rm dumb-init_*.deb
 
-# Install cget
-#RUN pip install cget
-
-# Install rclone
-#RUN pip install https://github.com/pfultz2/rclone/archive/master.tar.gz
 
 ARG MIOPEN_DIR=/root/dMIOpen
 #Clone MIOpen
 RUN git clone https://github.com/ROCmSoftwarePlatform/MIOpen.git $MIOPEN_DIR
 WORKDIR $MIOPEN_DIR
-ARG MIOPEN_BRANCH=b5c9cd5b0fa65bc77004dd59adcbb336ead031af
+ARG MIOPEN_BRANCH=38a0ebfd4a6b2e3eb356a09721a199f11d4a9a37
 RUN git pull && git checkout $MIOPEN_BRANCH
 
 ARG PREFIX=/opt/rocm
 ARG MIOPEN_DEPS=/opt/rocm
 
 # Install dependencies # included in rocm/miopen:ci_xxxxxx
-#RUN cmake -P install_deps.cmake --prefix $MIOPEN_DEPS
-#RUN CXXFLAGS='-isystem $PREFIX/include' cget install -f ./mlir-requirements.txt
+RUN . /env; if [ -z $NO_ROCM_INST ]; then\
+        pip install cget; \
+        pip install https://github.com/pfultz2/rclone/archive/master.tar.gz; \
+        cmake -P install_deps.cmake --prefix cget; \
+        cp -r cget/* /opt/rocm/.; \
```

Comment:
When I built this version of the dockerfile successfully I switched lines 109 and 110